circt/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM alpine
```

Comment:
Given that the llvm/circt builds are based on Ubuntu 20.04, we should probably stick with that. Actually, why not base this on the integration image? That way we'd get versions of Verilator, capnp, and yosys which have already been "validated" by the circt integration tests.

Reply:
Sure, I think that's fine. I had originally thought that this would be a minimal `docker` wrapper around CIRCT utilities, but it's fine if it's beefier. I was going to use this for other GitHub CI and I think packaging everything together (including Verilator) will work better there anyway.